tools/c7n_azure/c7n_azure/resources/network_security_group.py

```diff
@@ -39,9 +40,9 @@ class resource_type(object):
         )
 
 
-FROM_PORT = 'fromPort'
-TO_PORT = 'toPort'
+DIRECTION = 'direction'
 PORTS = 'ports'
+PORTS_OP = 'ports-op'
```

Comment:
To me ports_op would mean an operation like open or close, and not a check if the filter fulfills all ports or any of the ports. Maybe we could rename to MATCH ? So like MATCH 'all' or 'any' ?